### ISSUE TITLE
This package is quite large

### ISSUE THREAD

**Issue by mprast** [bee-queue/arena#383]
Hi all - I'm really glad that `bee-queue` has a full-featured monitoring dashboard and I don't want to look this particular gift horse in the mouth, but at a [14.5 MB install size](https://packagephobia.com/result?p=bull-arena) `bull-arena` is one of my largest dependencies (it might be the largest in fact). Truthfully I wasn't expecting arena to be this heavyweight, especially since it looks like it features embedding as a first-class use case, and since `bee-queue` has such an emphasis on size and simplicity. 

Has there been any thought on how to make this package smaller? Here are a few things that jumped out at me looking at your `package.json`:

- `lodash` looks like it's used in only a handful of places; using [per-method packages](https://lodash.com/per-method-packages) instead of a bulk import would be a bit more work but is probably be a pretty cost-effective way to save some space.
- if you're committed to `handlebars` I understand, but if not you may want to consider using a lighter-weight view framework. `eta` or `preact` come to mind but I'm sure there are many others. 
- in the same vein, `dayjs` is a smaller alternative to `moment` that you may want to look into.

Happy to submit a PR for any of these if you decide any one of them are worth pursuing, and of course feel free to ignore this if work is already being done to make arena more compact, or if there are good reasons why it can't be smaller. Thanks for taking a look!

**Comment by bradvogel** [comment]
You're right - this package is very large for what it does. Your ideas are great. PRs would be appreciated!
